sdk/resourcemanager/api-specs.json

```diff
@@ -24,12 +24,25 @@
     "args": "--payload-flattening-threshold=1 --tag=package-2021-02 --add-inner=AppServiceCertificate --name-for-ungrouped-operations=ResourceProvider",
     "note": "Add status code '200' to 'syncFunctionTriggers'"
   },
+  "appservice-hybrid": {
```

Comment:
It is easier for you to have the api-specs.json in resourcemanagerhybrid folder?

Reply:
Yes I had placed it there for using gulp. I considered placing these in a new `api-specs.json` in the resourcemanagerhybrid folder as well and adding a command-line option to specify which api-spec file should be used for doing codegen as well but didn't implement it.